config/srcSchemaMap.py

```diff
@@ -0,0 +1,209 @@
+config.srcSchemaMap = {
+"deblend_nChild": "deblend_nchild",
```

Comment:
I'm surprised by this, as I thought we'd be mapping from HSC field names with `.`s to LSST field names with `_`s, but clearly we're not.